### PR TITLE
Lasers now evaporate blood.

### PR DESCRIPTION
**Pull request by NonQueueingMatt**
As per title.

### CODE REVIEW

**Comment by NonQueueingMatt** [comment]
!review

**Comment by Kaedwuff** [comment]
It's a little bit of a nitpick, but your description of this PR does not include that you're changing the backend code for a genetics/wizard ability.

File: html/changelogs/mattatlas-laserfix.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes: 
+  - rscadd: "Burns now evaporate people's blood. Lasers evaporate less as they are more concentrated, while wide burns don't."
```
Comment (Geevies) [inline html/changelogs/mattatlas-laserfix.yml]:
```suggestion
  - rscadd: "Burns now evaporate people's blood. Lasers evaporate less as they are more concentrated, while wide burns evaporate more."
```

**Comment by NonQueueingMatt** [comment]
> 
> 
> It's a little bit of a nitpick, but your description of this PR does not include that you're changing the backend code for a genetics/wizard ability.

There is no backend code change. It's a define's name getting changed. It has absolutely no impact on anything.

**Pull request merged**